### DISCUSSION TITLE
SVD not working in the last version?

### DISCUSSION THREAD

**Discussion by AlejoJavierPonce** [lllyasviel/stable-diffusion-webui-forge#1231] [Q&A]
I`ve dropped these files (https://huggingface.co/stabilityai/stable-video-diffusion-img2vid-xt/tree/main) in the models/svd folder but the svd tab do not apper. Maybe I am missing something in the installation  or the svd feature do not work with the last version?

**Comment by Javagaga** [comment]
same situation, someone says it's been disabled temporally

**Comment by samueluchiha** [comment]
> I`ve dropped these files (https://huggingface.co/stabilityai/stable-video-diffusion-img2vid-xt/tree/main) in the models/svd folder but the svd tab do not apper. Maybe I am missing something in the installation or the svd feature do not work with the last version?

same situation here! any solution?
